src/libhirte/common/cfg.c

```diff
@@ -9,8 +9,13 @@
 #include <unistd.h>
 
 #include "libhirte/common/list.h"
+#include "libhirte/common/parse-util.h"
 #include "libhirte/hashmap/hashmap.h"
 #include "libhirte/ini/ini.h"
+#include "libhirte/log/log.h"
+
+#include "agent/agent.h"
```

Comment:
The dependency should be only `agent -> cfg` - this adds a dependency cycle. Same for the manager.

Reply:
we need the headers :-/ Decouple the heards in a common area would be nice.